src/lib/components/Molecule3dViewer.js

```diff
@@ -88,16 +88,14 @@ Molecule3dViewer.propTypes = {
      * Labels corresponding to the atoms of the molecule
      */
     labels: PropTypes.arrayOf(
-        PropTypes.exact({
+        PropTypes.shape({
             /**
              * The label content
              */
             text: PropTypes.string,
-
             /**
-             * Additional styling options. See https://dash.plotly.com/dash-bio/molecule3dviewer for details
+             * For additional styling keys, see https://3dmol.csb.pitt.edu/doc/types.html#LabelSpec
```

Comment:
OIC, it's not `text` and `parameters`, it's `text` and 17 other possible keys to specify the other parameters. I misunderstood that when I first looked at this PR. A comment here (with no following key) won't make it into the auto docs (as you can see from the built Py and R artifacts above, and the docs you DO get in this case are going to look funny if you specify `text` explicitly without specifying all the other ones (which I don't think we should do) - you can add a note about "see <docs> for all the other keys you can include" but it'll still say "labels has the following type: list of dicts containing keys 'text'." which would confuse people about the other keys.

So I guess really the best option here would be to go back toward what you originally had, something like:

```js
/**
 * Labels corresponding to the atoms of the molecule.
 * Each label has a `text` field, a string containing the label content,
 * and can have many other styling fields as described in
 * https://3dmol.csb.pitt.edu/doc/types.html#LabelSpec
 */
labels: PropTypes.arrayOf(PropTypes.object))
```

Reply:
Updated in cb6faa7.

Reply:
thx - and apologies for the whiplash 😅